astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 1ecac98d547729393ca54bcbe4f03cd4e1bfcd9a
+Subproject commit cca08a1755e703f9ff94e1f60a5b15d9521c128f
```

Comment:
@patti You committed a change of the `astropy_helpers` git submodule pointer in
https://github.com/patti/astropy/commit/dd9594361b9af480272d09a93aeb9a78a0d6718b

I guess this was accidental and needs to be removed?
Maybe @embray or @astrofrog can advise how to get rid of this change in this branch?
Is simply making a new commit that changes it back acceptable?
Or should @patti modify the previous commit to not include that change? (this is quite tricky, no?)